.jshintrc

```diff
@@ -1,28 +1,68 @@
 {
-    "browser": true,
+    "bitwise": false,
```

Comment:
you're right, it's probably not helping anything. our bottleneck is the DOM, not math.